c2_python-operating-system/2_managing-files-with-python/practice-quiz-managing-files-and-directories.md

```diff
@@ -82,7 +82,7 @@ import datetime
 def file_date(filename):
   # Create the file in the current directory
   with open (filename,'w') as file:
-    pass
+    {}
```

Comment:
can you explain what is this?

Reply:
Basically, it was that modified syntax that made the answer correct. Perhaps its because of some sort of syntax difference between the year you worked on, and the year I worked it on.